src/index.js

```diff
@@ -79,7 +80,7 @@ function ParseServer({
   cloud,
   collectionPrefix = '',
   clientKey = '',
-  javascriptKey = '',
+  javascriptKey = randomString(20),
```

Comment:
We still need to fix this a different way, like passing an unused key to the SDK.  We can't require client keys, and this change will do that..